index.js

```diff
@@ -0,0 +1,152 @@
+        tripList.append(`<li class='${trip.id}'>${trip.name}</li>`);
+      });
+    })
+    .catch(error => {
+      reportStatus(
+        `Encountered an error while loading trips: ${error.message}`
+      );
+      console.log(error);
+    });
+};
+
+// This gets the trip details given the id as a parameter
+const detailsTrips = id => {
+  reportStatus("Loading details...");
+
+  const tripDetailsList = $("#trip-details");
+  tripDetailsList.empty();
+
+  axios
+    .get(getURL)
+    .then(response => {
+      reportStatus(`Successfully loaded ${response.data.length} trips`);
+      // callback function which takes the id passed to it and finds the trip associated with that id
+      const trip = response.data.find(item => {
+        // returns true when that id matches the id passed to this method
+        return item.id.toString() === id;
+      });
+      loadForm(trip);
+      tripDetailsList.append(
+        `<h2>TRIP DETAILS</h2>`,
+        `<li>ID: ${trip.id}</li>`,
+        `<li>NAME: ${trip.name}</li>`,
+        `<li>CONTINENT: ${trip.continent}</li>`,
+        `<li>CATEGORY: ${trip.category}</li>`,
+        `<li>DURATION ${trip.weeks} weeks</li>`,
+        `<li>COST $${trip.cost.toFixed(2)}</li>`,
+        `<li>ABOUT: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. '</li>`
+      );
+    })
+    .catch(error => {
+      reportStatus(
+        `Encountered an error while loading trips: ${error.message}`
+      );
+      console.log(error);
+    });
+};
+
+// Loads the form and calls the submit fuction
+const loadForm = trip => {
+  reportStatus("Loading details...");
+  const tripReservation = $(".reserve-trip-area");
+  tripReservation.empty();
+
+  tripReservation.append(
+    `<h2>Reserve a Spot on ${trip.name}</h2>
+    <form class="reservation-form"> Name:<br />
+      <input class="text-input" type="text" name="name" value="Mickey" />
+      <br />
+      Email:<br />
+      <input class="email-input" type="text" name="email" value="mickey@mouse.com" />
+      <br /><br />
+      <input class="submit-click" type="submit" value="Submit" />
+    </form>`
+  );
+  setUpFormSubmission(trip);
+};
+
+const postForm = (name, email, id) => {
+  event.preventDefault();
+  axios({
+    method: "post",
+    url: `https://trektravel.herokuapp.com/trips/${id}/reservations`,
+    data: {
+      name: name,
+      email: email
+    }
+  })
+    .then(() => {
+      reportStatus(`Successfully submitted reservation`);
+    })
+    .catch(error => {
+      if (error.response.data && error.response.data.errors) {
+        reportError(
+          `Encountered an error while loading trips: ${error.message}`,
+          error.response.data.errors
+        );
+      } else {
+        reportStatus(`Encountered an error: ${error.message}`);
+      }
+    });
+};
+
+const listTrips = () => {
+  $("#load").click(getTrips);
+};
+
+const loadTripDetails = () => {
```

Comment:
The `loadTrip` and `loadTripDetails` functions don't really do those things directly, they just set up event handlers to do them eventually. It would be good if their names reflected that.